libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -210,6 +216,121 @@ describe("Drilling", () => {
                 .hasCellValue(0, 0, "$12,076,034.56")
                 .isCellUnderlined(DIRECT_SALES, false);
         });
+
+        it("Should drill down on heat map chart", () => {
+            const heatmapInsights = [
+                {
+                    title: "heat map only row measure",
+                    value: PRODUCT_VALUE,
+                    firstCell: INSIDE_SALES,
+                },
+                {
+                    title: "heat map only column measure",
+                    value: [
+                        "$15,582,695.69",
+                        "$16,188,138.24",
+                        "$30,029,658.14",
+                        "$5,863,972.18",
+                        "$5,763,242.30",
+                        "$6,978,618.41",
+                    ],
+                    firstCell: DIRECT_SALES,
+                },
+                {
+                    title: "heat map has measure column row",
+                    value: PRODUCT_VALUE,
+                    firstCell: INSIDE_SALES,
+                },
+            ];
+
+            Navigation.visit("dashboard/heatmap-drill-down");
+            heatmapInsights.forEach((insight, index) => {
+                new Widget(index)
+                    .scrollIntoView()
+                    .waitChartLoaded()
+                    .getChart()
+                    .waitLoaded()
+                    .hasTooltipContents(0, ["Department", "Amount"]);
```

Comment:
why don't chain here? .clickFirstCell();

Reply:
clickFirstCell() does not exist on type 'void'.

Reply:
I'll fix it